subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -684,6 +685,7 @@ u8_t ll_adv_enable(u8_t enable)
 
 #if defined(CONFIG_BT_CTLR_PHY)
 		conn->llcp_phy.req = conn->llcp_phy.ack = 0;
+		conn->llcp_phy.disabled = 0;
```

Comment:
Missing U?

Reply:
Updated.

Note: there are other cases where U is missing. You might want to got over it.
I really think this part of the code could benefit from a memset to zero.